conans/client/graph/graph_binaries.py

```diff
@@ -187,16 +188,19 @@ def _evaluate_node(self, node, build_mode, update, remotes):
                         # NO Build mode
                         self._process_node(node, pref, compatible_build_mode, update, remotes)
                         assert node.binary is not None
-                        if node.binary != BINARY_MISSING:
+                        if node.binary not in BINARY_MISSING:
```

Comment:
It works, but IMO it is less surprising to use a list/tuple and it is less error-prone, we might introduce a new label that is a subset of the string behind `BINARY_MISSING`:

```suggestion
                        if node.binary not in (BINARY_MISSING, ):
```

(or back to the `!=` operator)

Reply:
Yep, this was a bug, good catch!